Author: Travis Watkins <amaranth@ubuntu.com>
Description: keep window on top of fullscreen windows during move
Origin: vendor, ubuntu (1:0.7.2-0ubuntu2)
--- compiz.orig/plugins/move.c
+++ compiz/plugins/move.c
@@ -124,6 +124,11 @@ moveInitiate (CompDisplay     *d,
     xid = getIntOptionNamed (option, nOption, "window", 0);
 
     w = findWindowAtDisplay (d, xid);
+
+    /* make window stay above fullscreen windows while moving */
+    if (w)
+	updateWindowAttributes (w, CompStackingUpdateModeAboveFullscreen);
+
     if (w && (w->actions & CompWindowActionMoveMask))
     {
 	XRectangle   workArea;
